### PR TITLE
keyboard: backspace repeat

### PR DESCRIPTION
**Pull request by sshane**
- [x] key repeat
- [ ] fix sticky buttons

### CODE REVIEW

**Comment by pd0wm** [comment]
I sometimes hit a key, the touch down registers so the button changes color. However, it seems there is no corresponding touch up so the character never shows up and the button stays pressed.

File: selfdrive/ui/qt/widgets/keyboard.cc [hunk 1]
```diff
@@ -66,7 +66,7 @@ Keyboard::Keyboard(QWidget *parent) : QFrame(parent) {
     {"q","w","e","r","t","y","u","i","o","p"},
     {"a","s","d","f","g","h","j","k","l"},
     {"⇧","z","x","c","v","b","n","m","⌫"},
-    {"123","  ","⏎"},
+    {"123",",","  ",".","⏎"},
```
Comment (adeebshihadeh) [inline selfdrive/ui/qt/widgets/keyboard.cc:69]: revert these changes, let's stick to the ios layout
